dev-python/argon2_cffi/argon2_cffi-18.3.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} pypy pypy3 )
+inherit distutils-r1
+
+DESCRIPTION="CFFI bindings to the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2_cffi"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+CDEPEND="
```

Comment:
Please don't name it CDEPEND. We might be crazy enough to use CDEPEND for something else in a future EAPi, and it's easy to miss the accidental collision.

Reply:
Fair point. I used to write `RDEPEND="<python deps>"` and `DEPEND="${RDEPEND} <build deps>"`. Other devs disliked that version as well. Is there any good practice you can recommend for Python ebuilds?

Reply:
I have no clue why anybody would oppose using RDEPEND+DEPEND as you said. Generally if one is subset of the other, declare the smaller one first, then use it in the other. If dep groups aren't exact subset, either use `COMMON_DEPEND` or something like `DEPEND="..."; RDEPEND="${DEPEND} ..."; DEPEND+="..."`.